WarOfEmpires.QueryHandlers/Empires/GetSiegeQueryHandler.cs

```diff
@@ -31,17 +32,20 @@ public class GetSiegeQueryHandler : IQueryHandler<GetSiegeQuery, SiegeModel> {
                 TotalMaintenance = player.GetWorkerCount(WorkerType.SiegeEngineers) * player.GetBuildingBonus(BuildingType.SiegeFactory),
                 AvailableMaintenance = player.GetWorkerCount(WorkerType.SiegeEngineers) * player.GetBuildingBonus(BuildingType.SiegeFactory)
                     - player.SiegeWeapons.Sum(s => s.Count * SiegeWeaponDefinitionFactory.Get(s.Type).Maintenance),
-                FireArrowsInfo = MapSiegeWeapon(player, SiegeWeaponType.FireArrows),
-                BatteringRamsInfo = MapSiegeWeapon(player, SiegeWeaponType.BatteringRams),
-                ScalingLaddersInfo = MapSiegeWeapon(player, SiegeWeaponType.ScalingLadders)
+                SiegeWeapons = new List<SiegeWeaponModel>() {
+                    MapSiegeWeapon(player, SiegeWeaponType.FireArrows),
+                    MapSiegeWeapon(player, SiegeWeaponType.BatteringRams),
+                    MapSiegeWeapon(player, SiegeWeaponType.ScalingLadders)
+                }
             };
         }
 
-        private SiegeWeaponInfoViewModel MapSiegeWeapon(Player player, SiegeWeaponType type) {
+        private SiegeWeaponModel MapSiegeWeapon(Player player, SiegeWeaponType type) {
             var definition = SiegeWeaponDefinitionFactory.Get(type);
             var count = player.GetSiegeWeaponCount(type);
 
-            return new SiegeWeaponInfoViewModel() {
+            return new SiegeWeaponModel() {
+                Type = definition.Type.ToString("G"),
```

Comment:
According to your recent research, the ("G") can be removed here

Reply:
It's gone!